File: Exercicios_guanaraba/Com_Dart/bin/ex007_Media_aritimetica.dart
// Desenvolva um programa que leia as duas notas de um aluno, calcule e mostre a sua média.

import 'dart:io';

void main(){


print ('Nota 1: ');
int nota1 = int.parse(stdin.readLineSync());
print ('Nora 2: ');
int nota2 = int.parse(stdin.readLineSync());

print ('A média é: ' + ((nota1 + nota2) / 2).toString());



}

File: Exercicios_guanaraba/Com_Ruby/exercicios/ex007_Media_aritimetica.rb
#Desenvolva um programa que leia as duas notas de um aluno, calcule e mostre a sua média.

puts 'Nota 1: '
n1 = gets.chomp.to_f
puts 'Nota 2: '
n2 = gets.chomp.to_f

puts "A média das notas é : #{(n1+n2)/2}"